core/core_test.go

```diff
@@ -26,7 +27,7 @@ func TestRPCInvoker_InvokeFailinChainInit(t *testing.T) {
 	config.GlobalDefinition = &model.GlobalCfg{}
 	invoker := core.NewRPCInvoker(core.ChainName(""))
 	replyOne := &helloworld.HelloReply{}
-	ctx := metadata.NewContext(context.Background(), map[string]string{
+	ctx := context.WithValue(context.Background(), highway.AttachmentsKey{}, map[string]string{
```

Comment:
highway.AttachmentsKey{} 
rest must not call highway package, move this method to util package